client/deploy.go

```diff
@@ -1,46 +1,45 @@
 package client
 
 import (
+	"fmt"
+
 	"github.com/quintilesims/layer0/common/models"
 )
 
-func (c *APIClient) CreateDeploy(name string, content []byte) (*models.Deploy, error) {
-	req := models.CreateDeployRequest{
-		DeployName: name,
-		Dockerrun:  content,
-	}
-
-	var deploy *models.Deploy
-	if err := c.Execute(c.Sling("deploy").Post("").BodyJSON(req), &deploy); err != nil {
-		return nil, err
+func (c *APIClient) CreateDeploy(req models.CreateDeployRequest) (string, error) {
```

Comment:
This is an interesting change. What's the advantage of not passing back the entity models?

Reply:
This reflects the change in the API where any create/update/delete returns a job id